docs/migrating-to-velero.md

```diff
@@ -0,0 +1,50 @@
+# Migrating from Ark to Velero
+
+Users of Ark will need to [upgrade to v0.10.x][1], then move their installation to the renamed Velero images and Custom Resource Definitions (CRDs), as well as switch to the `velero` client. This is a one-time operation.
```

Comment:
I'd like to highlight clearly here that the scope of changes are: (1) CLI name change; (2) namespace & service account change; (3) image name change; and (4) API group name change.  Can we re-jigger this first paragraph to call that out, and then follow it with Prerequisites?

Reply:
That's what the list of high level changes was meant to accomplish. Do you think it would be clearer to merge that and the first paragraph?

Reply:
I think it's worth first calling out *what* is going to change (i.e. the 4 things i mentioned), and then separately summarizing *how* the migration will proceed (i.e. your list of steps).